src/api/routes/core-node-rpc-proxy.ts

```diff
@@ -166,12 +108,25 @@ export function createCoreNodeRpcProxyRouter(db: PgStore): express.Router {
     }
   }
 
-  router.post(
-    '/transactions',
-    asyncHandler(async (req, res, next) => {
+  const maxBodySize = 10_000_000; // 10 MB max POST body size
```

Comment:
I know we had this cap in the past but should we have this be configurable just in case?

Reply:
IIRC this is pretty much only here to override the default cap (1MB I think). But yeah probably still useful to make configurable. Maybe a good thing to do in a PR where we migrate to `env-schema` (I left that refactor out of this PR because of the already huge change set)

Reply:
Sounds good to me

Reply:
https://github.com/hirosystems/stacks-blockchain-api/issues/2055